### DISCUSSION TITLE
Provider installer pages link to incorrect page

### DISCUSSION THREAD

**Discussion by sebbASF** [apache/airflow#19575] [General]
### Describe the issue with documentation

https://airflow.apache.org/docs/apache-airflow-providers-airbyte/stable/installing-providers-from-sources.html has the following text:

"The packages are available via the Official Apache Software Foundations Downloads"

This links to http://ws.apache.org/mirrors.cgi which is not a valid download page, and anyway it belongs to the Web Services project.

I don't think the sentence adds any value, so could just be dropped.

### How to solve the problem

As per above, just remove the sentence

### Anything else

Note that this presumably applies to all the provider installation pages.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
I disagree with that assessment. This has been deliberately added there as a way to promote ASF/Downloads mechanism. It's not only a downoad page but also educational content to teach people about the ASF and the mechanism it uses. The page is ideally placed there - it explains people the purpose, of the page, explains that they also can install Airlfow using PyPI (as convenience packages) and when they are supposed to use officially downloaded sources. 

So I think it serves a very good purpose - especially if you empathethically think about users who land at this page who are different - they have no context, knowledge, experience with ASF mechanism. It is - I think - an important part  of communication: to provide more complex for people who want to explore it. And HTML (H stands for Hyperlink) make such additional context to be available as hyperlinked pages to the world-wide-web as a foundational element of WWW. 

I do not see it as a clear "issue" with the documentaiton - so @sebb If you still want to discuss (and maybe others have a different opinion there) I will convert it to discussion, and if others will express their opinions there and you would like to convince the others - feel free. I am happy to reconsider my opinion if there are others who think similarly.
